docs/http.md

```diff
@@ -9,37 +9,45 @@
 ## 属性
 | 属性 | 类型 | 是否必须 | 备注 |
 |--------|--------|--------|--------|
-|   url   | [expr](datatype.md)  | √ |  发起网络请求的目标地址，可以是http或https开头 |
-|   media   | [expr](datatype.md)  | x  |  post数据的媒体类型，默认为`text/plain`,可参考[MIME类型](https://developer.mozilla.org/zh-CN/docs/Web/HTTP/Basics_of_HTTP/MIME_types) |
-|   post   | [expr](datatype.md)  | x |  |
+|   url   | [string](datatype.md)  | √ |  发起网络请求的目标地址，可以是http或https开头 |
+|   media   | [string](datatype.md)  | x  |  post数据的媒体类型，默认为`text/plain`,可参考[MIME类型](https://developer.mozilla.org/zh-CN/docs/Web/HTTP/Basics_of_HTTP/MIME_types) |
+|   post   | [expr](datatype.md)  | x | http请求体 |
 |   headers   | [expr](datatype.md)  |  x |  头信息 可参考[HTTP Headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers) |
 |   async   | [boolean](datatype.md)  |  x | Default = $false  |
 |   path   | [path](datatype.md)  |  x | 返回结果存储路径  |
 
 ## 用法
 ### get请求
 ```yaml
-- http: {url: '"http://127.0.0.1/echo"', path: "/resp"}
+- http: {url: '"https://www.baidu.com?search=hello"', path: '/resp'}
+- echo: '$/resp'
 ```
 
 ### post请求
 ```yaml
-- http: {url: '"http://127.0.0.1/echo"', post: '"ping"' path: "/resp"}
+- http: {url: '"https://www.baidu.com"', path: '/resp', post: '"plain text body"'}
+- echo: '$/resp'
 ```
 
 ### 定义media的post请求
 ```yaml
+- http: {url: '"http://127.0.0.1/formPost"', media: '"application/x-www-form-urlencoded"', post: '"content=hello"', path: '/resp'}
```

Comment:
Try to get a reachable url to give the examples.
Maybe like that

- http: {url: '"https://reqbin.com/echo/post/form"', media: '"application/x-www-form-urlencoded"', post: '"content=hello"', path: '/resp'}
- echo: '$/resp'